controller/linodemachine_controller.go

```diff
@@ -49,6 +49,9 @@ import (
 	"github.com/linode/cluster-api-provider-linode/cloud/services"
 )
 
+// default etcd Disk size in MB
+const defaultEtcdDiskSize = 8192
```

Comment:
I don't think this is enough.

the max etcd data size *on-disk* is 8GiB. However, there is filesystem overhead, etc. so surely the disk itself must be larger than the etcd on-disk size?

Reply:
What would be your suggestion for a size if not 8 GB? If we allow nanodes for instance type on control plane nodes we have at most 25 GB for all disks attached to the instance (root, swap, and etcd).